scripts/single-table-migration-e2e.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -euxo pipefail
+
+rm migrate_e2e.sqlite || true
+
+bash <(curl https://raw.githubusercontent.com/ory/keto/master/install.sh) -b . v0.6.0-alpha.3
+
+export DSN="sqlite://./migrate_e2e.sqlite?_fk=true"
+export KETO_READ_REMOTE="127.0.0.1:4466"
+export KETO_WRITE_REMOTE="127.0.0.1:4467"
+
+./keto migrate up -y -c ./contrib/cat-videos-example/keto.yml
+./keto namespace migrate up -y -c ./contrib/cat-videos-example/keto.yml videos
+
+./keto serve all -c ./contrib/cat-videos-example/keto.yml &
+keto_server_pid=$!
+
+function teardown() {
+    kill $keto_server_pid || true
+}
+trap teardown EXIT
+
+jq '[range(200)] | map({namespace: "videos", object: . | tostring, relation: "view", subject: "user"})' <(echo '{}') \
```

Comment:
Should we create objects in multiple namespaces to ensure that all namespaces are migrated?

Reply:
True :+1:

Reply:
Is this already resolved? If not please resolve after they have been resolved in code :)

Reply:
I have it  resolved but not yet pushed :sweat_smile: CCI was too busy already yesterday for intermediate commits.